packages/nextjs/src/config/webpack.ts

```diff
@@ -242,17 +242,18 @@ function getWebpackPluginOptions(
   buildContext: BuildContext,
   userPluginOptions: Partial<SentryWebpackPluginOptions>,
 ): SentryWebpackPluginOptions {
-  const { isServer, dir: projectDir, buildId, dev: isDev, config: nextConfig } = buildContext;
+  const { isServer, dir: projectDir, buildId, dev: isDev, config: nextConfig, webpack } = buildContext;
 
+  const isWebpack5 = webpack.version.startsWith('5');
```

Comment:
I wonder if we an use the webpack 5 check to simplify the logic on how we add entry points with `addFileToExistingEntryPoint` (instead of having the various if else conditions)

Reply:
LOL - how these things come around. Long, long ago I actually had a version of the entry-adding code which did just that. I wonder if I still have it stashed somewhere...